pytest_cpp/plugin.py

```diff
@@ -1,16 +1,17 @@
-import fnmatch
 import os
 import stat
+
 import pytest
-from pytest_cpp.boost import BoostTestFacade
 
+from pytest_cpp.boost import BoostTestFacade
 from pytest_cpp.error import CppFailureRepr, CppFailureError
-
 from pytest_cpp.google import GoogleTestFacade
 
 FACADES = [GoogleTestFacade, BoostTestFacade]
 DEFAULT_MASKS = ('test_*', '*_test')
 
+_ARGUMENTS = 'arguments'
```

Comment:
How about calling this `cpp_arguments` instead? I feel `arguments` is too generic and doesn't indicate well which plugin uses the option.

Reply:
Agree

Reply:
Sure, I'll update README.
Thank you.

Reply:
Thanks!